js/entries/foundation.js

```diff
@@ -0,0 +1,101 @@
+import $ from 'jquery';
+
+import { Foundation } from '../foundation.core';
+Foundation.addToJquery($);
+
+// Add Foundation Utils to Foundation global namespace for backwards
+// compatibility.
+
+import { rtl, GetYoDigits, transitionend } from '../foundation.util.core';
+Foundation.rtl = rtl;
+Foundation.GetYoDigits = GetYoDigits;
+Foundation.transitionend = transitionend;
+
+import { Box } from '../foundation.util.box'
+import { onImagesLoaded } from '../foundation.util.imageLoader';
+import { Keyboard } from '../foundation.util.keyboard';
+import { MediaQuery } from '../foundation.util.mediaQuery';
+import { Motion, Move } from '../foundation.util.motion';
+import { Nest } from '../foundation.util.nest';
+import { Timer } from '../foundation.util.timer';
+
+Foundation.Box = Box;
+Foundation.onImagesLoaded = onImagesLoaded;
+Foundation.Keyboard = Keyboard;
+Foundation.MediaQuery = MediaQuery;
+Foundation.Motion = Motion;
+Foundation.Move = Move;
+Foundation.Nest = Nest;
+Foundation.Timer = Timer;
+
+// Touch and Triggers previously were almost purely sede effect driven,
+// so no // need to add it to Foundation, just init them.
+
+import { Touch } from '../foundation.util.touch';
```

Comment:
Ah gotcha, thanks for sending me that. I think personally I disagree with it because it makes things more verbose, but there's no need to reverse the decision. Works the same either way!